docs/global/modules/core/pages/reference/component-model-server.adoc

```diff
@@ -100,6 +100,6 @@ To enable it you can specify the following environment variables.
 
 The `permissions` claim of the token is expected to list directly granted permission.
 Additional granted permissions may be derived from that list.
-For example, an `admin` permission on a repository will grant write permission on all branches.
+For example, an `model-server/admin` permission on a repository will grant write permission on all branches.
```

Comment:
This is talking about admin permissions for a repository. e.g. `repository/someRepositoryId/admin`

Reply:
[f4b4161](https://github.com/modelix/modelix.core/pull/959/commits/f4b4161e9ad1948c657684af2602cb975bd29746) fixed this. Thank you for the fix!